cmd/api/src/api/v2/auth/auth.go

```diff
@@ -480,6 +480,23 @@ func (s ManagementResource) CreateUser(response http.ResponseWriter, request *ht
 				api.HandleDatabaseError(request, response, err)
```

Comment:
Let's get a return on the api.HandleDatabaseErrors, I noticed that a user still gets created when providing an invalid SAML ID